### DISCUSSION TITLE
Min and max options on dates

### DISCUSSION THREAD

**Discussion by u12206050** [directus/directus#18710] [Feature Requests]
### Summary

On the date interface it would be fantastic to be able to set the min and max dates that could be selected.
In addition these values can be derived from another local field on the same item, eg. `start_date` can not be after `end_date`
Could also support calculated dates such as `$NOW(-1 month)`

Previous closed PR that didn't have discussion: [10714](https://github.com/directus/directus/pull/10714)

### Motivation

We have a number of collections that have start and end dates and it has happened that these dates get added in wrong order.

### Detailed Design

![image](https://github.com/directus/directus/assets/6641242/dc4243ab-32d7-46ea-ad2a-3c9a19c85c1d)

### Must Have:
- min and max option for either static date or `$NOW`

### Should Have:
- min and max option where another local field could be specified
- Frontend validation

### Could Have:
- Field selection could be dropdown, with static dates and `$NOW` being added under "Other"

### Won't Have:
- Probably won't have server side validation at first

### Drawbacks

None

### Alternatives

Implement custom hook that checks those specific fields on create and update on the collections.

### Adoption Strategy

NA

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
